docs/concepts/ipfs-solves.md

```diff
@@ -0,0 +1,67 @@
+---
+title: IPFS and the problems it solves
+description:  Learn about the problems that IPFS solves.
+---
+
+# IPFS and the problems it solves
+
+IPFS seeks to address problems with the current web and existing data representation / routing / transfer protocols like <VueCustomTooltip label="A protocol for transferring data over the internet, mainly used for web browsing. It enables communication between a client (e.g. a web browser) and a server, where the client sends a request and the server returns a response with the requested information." underlined multiline is-medium>HTTP</VueCustomTooltip> including:
+
+- [Centralization](#centralization)
+- [Performance](#performance)
+- [Resilience](#resilience)
+- [Link rot](#link-rot)
+- [Verifiability](#verifiability)
+- [Data sovereignty and ownership](#data-sovereignty)
+- [Off-chain storage](#off-chain-storage)
+- [Local-first software](#local-first-software)
+- [Vendor lock-in](#vendor-lock-in)
+
+In this guide, you'll learn about each of the problems that IPFS solves.
+
+:::callout
+This guide is part 2 of a 3-part introduction to [the basic concepts of IPFS](../concepts/README.md#learn-the-basics). The third part, [**How IPFS Works**](../concepts/how-ipfs-works.md), covers the different subsystems that IPFS is comprised of and how each one works.
+:::
+
+## Centralization
```

Comment:
For consistency with the new IPFS website:

In the "Why IPFS?" section, we open with the following three reasons: **Open, Verifiable, and Resilient**

Can you adjust this page to be coherent with that?


- Emphasise that IPFS is an **open** protocol/network: "The IPFS network is distributed and participatory, which reduces data silos from central servers." in this paragraph. 
- Bring Verifiable and resilient to the top